src/context/directory/handlers/branding.ts

```diff
@@ -10,9 +10,14 @@ type ParsedBranding = {
 };
 
 function parse(context: DirectoryContext): ParsedBranding {
+  const brandingDirectory = path.join(context.filePath, constants.BRANDING_DIRECTORY);
+
+  if (!existsMustBeDir(brandingDirectory)) return { branding: undefined };
+
+  const branding = loadJSON(path.join(brandingDirectory, 'branding.json'), context.mappings);
```

Comment:
this will break a few integrations if the tenant doesn't have No-Code Editor feature enabled. `branding` directory might exist for Page Template feature, but the `branding.json` doesn't. If the file doesn't exists, this should return `{ branding: undefined }`.

Reply:
this is what happens on accounts that don't have this feature enable, but use universal login page template.
![image](https://user-images.githubusercontent.com/1906333/167176064-9e4ba2e8-3187-46b8-b108-dd646c536460.png)

Reply:
@tpires Please open a separate issue outlining the problem. Thanks.